hyperion/models/user.py

```diff
@@ -48,6 +48,12 @@ def __str__(self):
         else:
             return super().__str__()+'foreign_user'+str(self.display_name)
 
+    def get_full_id(self):
+        if self.host:
+            return str(self.host.name) + "/author/" + str(self.author.id)
```

Comment:
@Raymundo1 should use string templating API instead of manual string concatenation
```suggestion
            return '{}/author/{}'.format(self.host.name, self.author.id)
```

Reply:
@Raymundo1 also, `host` is nullable. Your should have handling for this case

Reply:
@Raymundo1 I don't think that the url relative method should be written in model class.
If I understand your code correctly, the **get_full_id** method is to return the unique url path, something like a path in **localhost:8080/server/author/1**, this method should be moved into some view  factory. What is your opinions?  @ExiaSR @zytwh2012

Reply:
我们打算把`host.name`弄成通过environment variable来设置

我觉得都行，不清楚django best practise是怎么样的